Src/libCZI/CziUtils.cpp

```diff
@@ -111,73 +111,74 @@ using namespace libCZI;
 /// <returns>	A boolean indicating whether the coordinates are equal or not (according to the above definition of equality). </returns>
 /*static*/bool CziUtils::CompareCoordinate(const libCZI::IDimCoordinate* coord1, const libCZI::IDimCoordinate* coord2)
 {
-		bool areEqual = true;
-		CziUtils::EnumAllCoordinateDimensions(
-				[&](libCZI::DimensionIndex dim)->bool
-				{
-						int coordinate1, coordinate2;
-						if (coord1->TryGetPosition(dim, &coordinate1) == true)
-						{
-								if (coord2->TryGetPosition(dim, &coordinate2) != true)
-								{
-										areEqual = false;
-										return false;
-								}
-
-								if (coordinate1 != coordinate2)
-								{
-										areEqual = false;
-										return false;
-								}
-						}
-
-						return true;
-				});
-
-		return areEqual;
+    bool areEqual = true;
+    CziUtils::EnumAllCoordinateDimensions(
+        [&](libCZI::DimensionIndex dim)->bool
+        {
+            int coordinate1, coordinate2;
+            if (coord1->TryGetPosition(dim, &coordinate1) == true)
+            {
+                if (coord2->TryGetPosition(dim, &coordinate2) != true)
+                {
+                    areEqual = false;
+                    return false;
+                }
+
+                if (coordinate1 != coordinate2)
+                {
+                    areEqual = false;
+                    return false;
+                }
+            }
+
+            return true;
+        });
+
+    return areEqual;
 }
 
 /*static*/void CziUtils::EnumAllCoordinateDimensions(std::function<bool(libCZI::DimensionIndex)> func)
 {
-		static const DimensionIndex dims[] = { DimensionIndex::Z,DimensionIndex::C,DimensionIndex::T,DimensionIndex::R,DimensionIndex::S,
-												DimensionIndex::I,DimensionIndex::H,DimensionIndex::V,DimensionIndex::B };
-
-		for (int i = 0; i < sizeof(dims) / sizeof(dims[0]); ++i)
-		{
-				if (func(dims[i]) == false)
-						break;
-		}
+    static const DimensionIndex dims[] = { DimensionIndex::Z,DimensionIndex::C,DimensionIndex::T,DimensionIndex::R,DimensionIndex::S,
+                                            DimensionIndex::I,DimensionIndex::H,DimensionIndex::V,DimensionIndex::B };
+
+    for (int i = 0; i < sizeof(dims) / sizeof(dims[0]); ++i)
+    {
+        if (func(dims[i]) == false)
+        {
+            break;
+        }
+    }
 }
```

Comment:
This looks like some massive flaw/bug in the past -> not having those brackets there before seems to kind of change logic completely, doesn't it?

Reply:
Not, no a flaw, just a case of "omitting braces for single nested statements" which is perfectly legal here, but I'd recommend "always have braces" (which is the convention which most of the codebase here complies with). As said - needs to go into some to-be-written "coding conventions"-section of the documentation.